lib/ansible/modules/identity/1password_facts.py

```diff
@@ -0,0 +1,313 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (c) 2018, Ryan Conway (@rylon)
+# (c) 2018, Scott Buchanan <sbuchanan@ri.pn> (onepassword.py used as starting point)
+# (c) 2018, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+# Ansible boilerplate: https://docs.ansible.com/ansible/2.6/dev_guide/testing/sanity/boilerplate.html
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+module: 1password_facts
```

Comment:
This should be name `onepassword_facts`. While I'm sure this works, there are many places in Python where using a digit at the beginning is unsupported, and we are asking for trouble with this name.

Reply:
Yeah good point, I'll rename it to be on the safe side.